lib/Dialect/OM/Evaluator/Evaluator.cpp

```diff
@@ -621,8 +647,7 @@ circt::om::Evaluator::evaluateListConcat(ListConcatOp op,
 
     // Append each EvaluatorValue from the sublist.
     evaluator::ListValue *subList = extractList(result.value().get());
-    for (auto subValue : subList->getElements())
-      values.push_back(subValue);
+    extractElements(subList);
```

Comment:
Does this assume that the concat argument is always a list of sublists? So this doesn't support concatenating two "depth 1" lists?

Reply:
yeah, this was just wrong.